### DISCUSSION TITLE
PDP - Media gallery vertical thumbnail

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#57] [1. Help]
Hi There,

I want to make the product image gallery vertical thumbnails slider on Product details page as below : 
How can I implement this in the Breeze theme?
![image](https://user-images.githubusercontent.com/9527439/214047588-819bfa0a-3565-4a96-bdbc-34cb00f6fbfb.png)

Can you please guide me?

Thanks
Rutul

**Comment by vovayatsyuk** [comment]
You can do this just like in Luma. Copy the values you need to change from blank theme into yours: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/etc/view.xml#L242-L248

**Comment by rutulritz** [comment]
Hi @vovayatsyuk ,

Thanks for the solution. 

Now I want to show the **single item** on carousel with dots  and arrow using this : https://breezefront.com/cms-elements#carousel-slider.

How can I implement this. Please guide me on this.

Thanks

**Reply by vovayatsyuk** [reply]
Sorry, I don't understand what you are trying to do, and how the carousel slider is related to image thumbnails.

Please show me the code you already wrote and maybe then I could help.

**Reply by rutulritz** [reply]
@vovayatsyuk no problem, I got it and fixed the same.

Thanks

**Comment by rutulritz** [comment]
Hi @vovayatsyuk ,
In mobile vertical thumbnail is not working properly.

**Reply by rutulritz** [reply]
Hi @vovayatsyuk 

Please help...

**Reply by vovayatsyuk** [reply]
Please describe your issue.

**Reply by rutulritz** [reply]
I want to add arrows to thumbnails as well same as main image on gallery

**Reply by vovayatsyuk** [reply]
Okay. What do mean saying “In mobile vertical thumbnail is not working properly.” and how it’s related to arrows?

**Reply by rutulritz** [reply]
in Mobile responsive thumbnail container is not showing vertically.

**Reply by vovayatsyuk** [reply]
It’s by design. Here is how we apply vertical mode for large screens: https://github.com/breezefront/module-breeze/blob/master/view/frontend/web/css/source/components/_gallery.less#L296

You can do the same in your theme’s styles, but for smaller screen.

In future I will use some “vertical-gallery-media” variable instead of screen__l so you can change just one variable to get vertical thumbs on target devices.
